ql/src/java/org/apache/hadoop/hive/ql/propertymodifier/Constants.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.hadoop.hive.ql.propertymodifier;
+
+/**
+ * Created by kartik.bhatia on 14/05/18.
+ */
+public class Constants {
+
+  public static String MAPRED_QUEUE_NAME = "mapreduce.job.queuename";
```

Comment:
These properties should be already available somewhere in hive code or in its mr dependency. Can you check>